iOS/BugsnagTests/Swift Tests/BugsnagMetadataTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  BugsnagMetadataTests.swift
+//  Tests
+//
+//  Created by Robin Macharg on 10/02/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+import XCTest
+
+class BugsnagMetadataTests: XCTestCase {
+
+    /**
+     * Test that getMetadata() is exposed to Swift correctly
+     */
+    func testgetMetadataIsExposedCorrectly() {
```

Comment:
My interpretation of the spec is that the `BugsnagMetadata` class itself doesn't need to be exposed, and that it's sufficient to simply specify the `addMetadata/clearMetadata/getMetadata` methods instead. Perhaps that's something we could check with the team before thinking of changing anything here?

Reply:
This - that the `BugsnagMetadata` class is only intended for internal (and therefore ObjC-only) use - has been confirmed in internal discussion.  I'll remove the `NS_SWIFT_NAME`s and associated tests.